legend-engine-xt-analytics-mapping-pure/src/main/java/org/finos/legend/pure/code/core/CoreExternalMappingAnalysisCodeRepositoryProvider.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Goldman Sachs
```

Comment:
I think you need to do something like https://github.com/finos/legend-engine/blob/master/legend-engine-xt-graphQL-pure/src/test/java/org/finos/legend/engine/code/core/Test_GraphQL_Pure.java to collect and run tests in pipeline